views/project/applicants.ejs

```diff
@@ -18,11 +18,48 @@
             function afterSort(){
                 addLink();
             }
+
+            //Used to copy columns of applicants table
+            $(document).ready(function() {
+                let tableData = $("#projectTbody tr");
+                
+                $("#copyButton").click(function() {
+                    let colsArray = [];
+                    $("#copyTable input").each(function(index) {
+                        if($(this).is(':checked')) {
+                            colsArray.push(index);
+                            $(this).prop('checked', false);
+                        }
+                    })
+                    if(colsArray.length > 0) {
+                        copyColumns(colsArray);
+                    }
+                })
+
+                function copyColumns(colsArray) {
+                    let $temp = $("<textarea>");
+                    let copyString = '';
+
```

Comment:
Also add another option (check box) to choose if copy the headers please, copy with multiple column really needs header